inc/thorvg.h

```diff
@@ -300,6 +300,51 @@ class TVG_EXPORT Paint
      */
     uint8_t opacity() const noexcept;
 
+    /**
+     * @brief Const forward iterator-like class enabling the iteration over the children nodes of the given paint.
+     *
+     * For the Scene-type Paint the children nodes represent the paints pushed into the Scene - the order of the children nodes is the same as the order as they were pushed. For the Picture-type Paint the child node is the read image in one of the supported formats. The Shape-type Paint doesn't have any child nodes.
+     *
+     * @BETA_API
+     */
+    class Iterator
+    {
+        const Paint* parent;
+        const Paint* child;
+
+        public:
+            Iterator (Paint* p = nullptr, Paint* c = nullptr);
+            const Paint& operator*() const;
+            Iterator& operator++();
+            Iterator operator++(int);
+            friend bool operator!=(const Iterator& it1, const Iterator& it2)
```

Comment:
I wasn't able to move these definitions (!= and ==) from this file. I'd prefer to put them together with the rest in the tvgPaint.cpp file